sql/2021/cdn/distribution_of_http_versions_cdn_vs_origin.sql

```diff
@@ -0,0 +1,69 @@
+#standardSQL
+# 17_19: Percentage of HTTPS responses by protocol
+SELECT
+  a.client,
+  IF(cdn = "ORIGIN", "ORIGIN", "CDN") AS cdn, firstHtml,
+  COUNTIF(IFNULL(a.protocol, b.protocol) = 'HTTP/0.9') AS http09,
+  COUNTIF(IFNULL(a.protocol, b.protocol) = 'HTTP/1.0') AS http10,
+  COUNTIF(IFNULL(a.protocol, b.protocol) = 'HTTP/1.1') AS http11,
+  COUNTIF(IFNULL(a.protocol, b.protocol) = 'HTTP/2') AS http2,
+  COUNTIF(IFNULL(a.protocol, b.protocol) = 'H3-29' OR IFNULL(a.protocol, b.protocol) = 'H3-Q050') AS http3,
```

Comment:
We haven't done this for HTTP chapter as it's unrealiable. A server that supports HTTP/3 will use HTTP/2 for the initial, "cold" request and then use "HTTP/3" for future requests. for that reason we've lumped HTTP/2 and HTTP/3 together as "HTTP/2+" to show it could be HTTP/2 (or better).

Reply:
@Navaneeth-akam  FYI ^

Reply:
@tunetheweb  - We updated the Google sheet to correct this.